include/simdjson/common_defs.h

```diff
@@ -168,6 +168,19 @@ constexpr size_t DEFAULT_MAX_DEPTH = 1024;
 
 #endif // MSC_VER
 
+#if defined(simdjson_inline)
+    // Prefer the user's definition of simdjson_inline; don't define it
+    // ourselves.
+#elif defined(__GNUC__) && !defined(__OPTIMIZE__)
```

Comment:
```suggestion
#elif defined(NDEBUG)
```

Reply:
Note that NDEBUG is part of the C/C++ standards, it is defined across compilers.